services/wallet/thirdparty/rarible/types.go

```diff
@@ -310,6 +311,13 @@ func raribleToCollectionsData(l []Collection, isMainnet bool) []thirdparty.Colle
 	return ret
 }
 
+func (c *Collection) toCollectionSocials() thirdparty.CollectionSocials {
```

Comment:
seems like we can't use Rarible for socials since the data they provide is incomplete. If you make this field a pointer just leave it as nil for this provider.